### DISCUSSION TITLE
Subnet under 24th

### DISCUSSION THREAD

**Discussion by DenisKrym** [wg-easy/wg-easy#1540] [Q&A] [answered]
Hi
Can I set the wireguard subnet address with a 26th netmask?

**Comment by fahadshery** [comment]
> Hi Can I set the wireguard subnet address with a 26th netmask?

yes, shouldn't be a problem as long as the range is valid

**Reply by DenisKrym** [reply]
Thanks for the reply!

How can this be specified? If I set “WG_DEFAULT_ADDRESS_RANGE=26”, nothing changes in the config.

If I put “WG_DEFAULT_ADDRESS=10.0.128.129/26”, then the config shows (Address = 10.0.128.129/24) and (POSTROUTING -s 10.0.128.129/26/24).

**Comment by ElDavoo** [comment, marked as answer]
Feature is planned for v15, see #1347 .
It's a pity that it's currently impossible to use this feature without building a custom image.

**Reply by DenisKrym** [reply]
Thanks for the reply!

Let's wait for the feature!

**Reply by ElDavoo** [reply]
I migrated to wireguard-ui as it supports all the advanced wg options that I need.
